packages/@ama-sdk/core/cli/files-pack.cts

```diff
@@ -27,7 +26,7 @@ const files = [
 /**  Update package.json exports */
 const updateExports = async () => {
   const packageJson = JSON.parse(await fs.readFile(path.join(baseDir, 'package.json'), { encoding: 'utf8' }));
-  const packageJsonFiles = globby.sync(path.posix.join(posixBaseDir, distFolder, '*', '**', 'package.json'), {absolute: true});
```

Comment:
don't we need this is `argv.cmd` is passed ?

Reply:
Let's open a ticket to follow up